src/Files.App.Controls/BreadcrumbBar/EventArgs.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace Files.App.Controls
 {
-	public record class BreadcrumbBarItemClickedEventArgs(BreadcrumbBarItem Item, int Index, bool IsRootItem = false);
+	public record class BreadcrumbBarItemClickedEventArgs(BreadcrumbBarItem Item, int Index, bool IsRootItem = false, bool IsMiddleButtonPressed = false);
```

Comment:
Instead of passing a Boolean, would it be possible to pass the pointer args directly?

Reply:
Done